src/toTransform.js

```diff
@@ -109,7 +123,8 @@
       convertTranslate(properties.translate),
       convertRotate(properties.rotate),
       convertScale(properties.scale),
-      convertOffsetAnchorPosition(properties, element)
+      convertOffsetAnchorPosition(properties, element),
+      convertOffsetRotate(properties, element)
```

Comment:
If we always add trailing commas, these patches become easier: typically one line changes instead of 2.